### ISSUE TITLE
can't run local server

### ISSUE THREAD

**Issue by boneskull** [jsperf/jsperf.com#65]
two reasons:

1.  setup script is invalid; needs a `'use strict';` at top

	```
	> jsperf.com@2.0.0 setup /Volumes/alien/projects/forked/jsperf.com
	> node setup

	/Volumes/alien/projects/forked/jsperf.com/setup/index.js:112
	  let prop;
	  ^^^

	SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
	    at exports.runInThisContext (vm.js:53:16)
	    at Module._compile (module.js:387:25)
	    at Object.Module._extensions..js (module.js:422:10)
	    at Module.load (module.js:357:32)
	    at Function.Module._load (module.js:314:12)
	    at Function.Module.runMain (module.js:447:10)
	    at startup (node.js:139:18)
	    at node.js:999:3
	```

2. ...and it's seemingly incompatible with mysql 5.7.10 via homebrew:

	```
	Connected to database...
	Successfully created database
	Granted permissions to your user
	Prepared to create tables
	/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/protocol/Parser.js:82
	        throw err;
	        ^

	Error: ER_COLLATION_CHARSET_MISMATCH: COLLATION 'utf8mb4_unicode_ci' is not valid for CHARACTER SET 'utf8'
	    at Query.Sequence._packetToError (/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/protocol/sequences/Sequence.js:48:14)
	    at Query.ErrorPacket (/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/protocol/sequences/Query.js:83:18)
	    at Protocol._parsePacket (/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/protocol/Protocol.js:271:23)
	    at Parser.write (/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/protocol/Parser.js:77:12)
	    at Protocol.write (/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/protocol/Protocol.js:39:16)
	    at Socket.<anonymous> (/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/Connection.js:96:28)
	    at emitOne (events.js:90:13)
	    at Socket.emit (events.js:182:7)
	    at readableAddChunk (_stream_readable.js:147:16)
	    at Socket.Readable.push (_stream_readable.js:111:10)
	    --------------------
	    at Protocol._enqueue (/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/protocol/Protocol.js:135:48)
	    at Connection.query (/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/Connection.js:201:25)
	    at /Volumes/alien/projects/forked/jsperf.com/setup/index.js:221:12
	    at Array.forEach (native)
	    at Handshake._callback (/Volumes/alien/projects/forked/jsperf.com/setup/index.js:218:36)
	    at Handshake.Sequence.end (/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/protocol/sequences/Sequence.js:96:24)
	    at Handshake.Sequence.OkPacket (/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/protocol/sequences/Sequence.js:105:8)
	    at Protocol._parsePacket (/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/protocol/Protocol.js:271:23)
	    at Parser.write (/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/protocol/Parser.js:77:12)
	    at Protocol.write (/Volumes/alien/projects/forked/jsperf.com/node_modules/mysql/lib/protocol/Protocol.js:39:16)
	```

**Comment by maxbeatty** [comment]
sorry for the trouble. I'll make a quick PR to include the changes from #59 to fix mysql and some changes in #58 to fix the setup script. will update this once it lands on master

**Comment by boneskull** [comment]
tyvm
